frontend/app/views/event/page.scala.html

```diff
@@ -172,19 +172,35 @@ <h1 class="event-masthead__name">@event.name.text</h1>
                                         </div>
                                     } else {
                                         @for(pricing <- ticket.cost) {
-                                            <div class="js-event-price event-content__item">
-                                                <div class="event-content__price event-content--line-divider js-event-price-value" data-discount-text="@pricing.discountPrice" itemprop="highPrice">
-                                                    @pricing.formattedPrice
+
+                                            @if(event.hasMemberTicket) {
+                                                <div class="js-event-price event-content__item">
+                                                    <div class="event-content__price event-content--line-divider js-event-price-value"
```

Comment:
Sure I started with this approach. I also want to take the .js-event-price class out of action so the check in js is just if this element exists. I also don't want a data-discount-text attribute sat in the markup for non discounted events. It was this or wrap if statements around those classes and attributes. This felt cleaner.